tasks/go_deps.py

```diff
@@ -115,3 +115,110 @@ def send_count_metrics(
         print(color_message("Sending metrics to Datadog", "blue"))
         send_metrics(series=series)
         print(color_message("Done", "green"))
+
+
+BINARY_TO_TEST = ["serverless"]
+
+
+@task
+def test_dependencies_list(
```

Comment:
A large chunk of both functions seems quite redundant (mostly the iteration logic and cmd launch prep), maybe it could be refactored into a separate function. I don't mind this much either.